dev/com.ibm.ws.ejbcontainer.management.j2ee_fat/fat/src/com/ibm/ws/ejbcontainer/management/j2ee/fat/tests/EJBJ2EEManagementTest.java

```diff
@@ -52,14 +53,23 @@ public class EJBJ2EEManagementTest {
     @Server("com.ibm.ws.ejbcontainer.management.j2ee.fat")
     public static LibertyServer server;
 
+    /**
+     * @ClassRule
+     *            public static RepeatTests r =
+     *            RepeatTests.with(FeatureReplacementAction.EE7_FEATURES().fullFATOnly().forServers("com.ibm.ws.ejbcontainer.management.j2ee.fat")).andWith(FeatureReplacementAction.EE8_FEATURES().forServers("com.ibm.ws.ejbcontainer.management.j2ee.fat"));
+     **/
     @ClassRule
-    public static RepeatTests r = RepeatTests.with(FeatureReplacementAction.EE7_FEATURES().fullFATOnly().forServers("com.ibm.ws.ejbcontainer.management.j2ee.fat")).andWith(FeatureReplacementAction.EE8_FEATURES().forServers("com.ibm.ws.ejbcontainer.management.j2ee.fat"));
```

Comment:
We should not disable the repeat of EE7_FEATURES here.  Instead we should add the EE7 repeat ID to the `addUnsupportedRepeatIDs` of the checkpoint rule.  You already added the `EmptyAction.ID` there, but I am unclear which run that is skipping since you updated the base server to use EE7 features.

Reply:
This still needs to be addressed.